public/app/features/explore/TableContainer.tsx

```diff
@@ -47,13 +47,13 @@ export class TableContainer extends PureComponent<TableContainerProps> {
     const tableWidth = width - config.theme.panelPadding * 2 - PANEL_BORDER;
     const hasTableResult = tableResult?.length;
 
-    if (hasTableResult) {
+    if (tableResult && tableResult.length) {
       // Bit of code smell here. We need to add links here to the frame modifying the frame on every render.
```

Comment:
interesting that  `tableResult?.length` throws error but `tableResult && tableResult.length` works.